hacks/isolateVersion/assets/README.md

```diff
@@ -0,0 +1,3 @@
+# What is this folder?
```

Comment:
It's kind of a weird folder, so I figured I should explain it.

Basically everything in here is going to be used in some way by the `isolateVersion` script. For this step, the file I added gets copied over to the `src/theme` directory.